utils/CameraPermissionUtils.js

```diff
@@ -1,4 +1,4 @@
-import PermissionsAndroid from 'react-native';
+import {PermissionsAndroid} from 'react-native';
```

Comment:
shouldn't be with spaces like this? 
import { PermissionsAndroid } from 'react-native';

Reply:
Yes, updated as mentioned.